src/main/scala/mesosphere/marathon/api/RestResource.scala

```diff
@@ -73,4 +74,15 @@ trait RestResource {
       case Success => fn(t)
     }
   }
+
+  def decodeBytes(data: Array[Byte], req: HttpServletRequest): String = {
+    val maybeEncoding = Option(req.getCharacterEncoding())
+    val charset = maybeEncoding match {
+      case Some(charsetName) =>
+        java.nio.charset.Charset.forName(charsetName)
+      case None =>
+        StandardCharsets.UTF_8 // TODO(jdef) should this be configurable somewhere?
```

Comment:
The HTTP spec says we should use ISO-8859-1 if it is not specified. Given that UTF-8 is a strict super set, I think its the right choice. And no, it shouldn't be configurable.